scripts/build.ps1

```diff
@@ -479,6 +449,71 @@ function PrintAndExit-OnError([System.String] $output){
     }
 }
 
+function Locate-MSBuildPath() {
+
+  $vsInstallPath = Locate-VsInstallPath
+  $msbuildPath = Join-Path -path $vsInstallPath -childPath "MSBuild\$env:MSBUILD_VERSION\Bin"
+
+  return Resolve-Path -path $msbuildPath
+}
+
+function Locate-VsInstallPath() {
+
+   $locateVsApi = Locate-LocateVsApi
+
+   $requiredPackageIds = @()
```

Comment:
If we know the list of elements, there is no need to append to the collection.

$`requiredPackageIds = @('a', 'b')`